src/aqdata/aq_base.h

```diff
@@ -5,7 +5,7 @@
 #include <map>
 #include <unordered_map>
 #include <string>
-
+#include <iomanip>
 #include <deal.II/base/tensor.h>
```

Comment:
Add a line between 8 and 9 as `iomanip` is STL while deal.II is third-party library.